### ISSUE TITLE
[Bug]: Always Error in IDE Logs

### ISSUE THREAD

**Issue by CyberXCoder-6649** [AndroidIDEOfficial/AndroidIDE#1114]
### What happened?

I have question:
This issue was from long time I didn't reported this issue, should I ignore issue 

See Relavent Log output always showing in IDE logs  

### What's the expected behavior?

.

### What version of AndroidIDE you're using?

v2.5.0 (debug builds)

### Relevant log output

```shell
ToolingApiErrorStream     E   SLF4J: No SLF4J providers were found. 
ToolingApiErrorStream     E   SLF4J: Defaulting to no-operation (NOP) logger implementation 
ToolingApiErrorStream     E   SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by itsaky** [comment]
This does not affect your project in any way and can be ignored.

**Comment by CyberXCoder-6649** [comment]
I am getting many ide crash should I report every crash I didn't report some crash I thought it will not be allowed so is it allowed to share all crash report of debug builds?

And I want to contribute to AndroidIDE with AndroidIDE how can I?

**Comment by itsaky** [comment]
> is it allowed to share all crash report of debug builds?

Yes, there is no such restriction on many issues you can create (create a separate issue for each bug/feature/suggestion). Just make sure that you look into the existing issues to see if that issue has already been reported or not. If you open a duplicate issue, it'll be marked as "duplicate" and closed immediately.

> And I want to contribute to AndroidIDE with AndroidIDE how can I? Means can I compile AndroidIDE project from AndroidIDE

Read the [contribution guidelines](https://github.com/AndroidIDEOfficial/AndroidIDE/blob/dev/CONTRIBUTING.md). Also, building the AndroidIDE project on an Android device can be a heavy task. I hope you know what you're doing.

**Comment by itsaky** [comment]
> Means can I compile AndroidIDE project from AndroidIDE

You may have a look at [this comment](https://github.com/AndroidIDEOfficial/AndroidIDE/issues/1096#issuecomment-1635788700).
